tools/generate_requirements.py

```diff
@@ -53,20 +53,53 @@ def generate_environment_yml(req_sections: dict[str, list[str]]) -> None:
 
             pkgname = re.split('[>=]', dep)[0]
             dep = dep.replace(pkgname, rename_idx.get(pkgname, pkgname))
+            if dep == "scikit-image":
+                continue
 
             lines.append(f"  - {dep}")
 
     with open("environment.yml", "w") as f:
         f.writelines(f"{line}\n" for line in lines)
 
 
+def explode_dependencies(
+    dep_list: list[str],
+    optional_dict: dict[str, list[str]],
+    package_name: str = 'scikit-image',
```

Comment:
This can't be used with anything other than 'scikit-image' presumably, so we can just hard-code.